src/components/Advanced.js

```diff
@@ -44,6 +45,7 @@ class Advanced extends Component {
     encrypt = () => {
         try {
             const decrypted = this.state.json;
+            console.log(decrypted);
```

Comment:
you'll probably want to remove this 😅

Reply:
I noticed that while doing some other work on it today. Oops!

I'll remove it.

Reply:
Fixed